resources/mw.widgets.EntitiesMultiselectWidget.js

```diff
@@ -88,12 +88,12 @@
 	mw.widgets.EntitiesMultiselectWidget.prototype.getOptionsFromData = function ( data ) {
 		const items = [];
 
-		for ( let i = 0; i < data.search.length; i++ ) {
+		data.search.forEach( function ( entity ) {
 			items.push( new OO.ui.MenuOptionWidget( {
-				data: data.search[ i ].id,
-				label: data.search[ i ].label
+				data: entity.id,
+				label: entity.label
 			} ) );
-		}
+		} );
```

Comment:
Why not `map`?

Reply:
I think I hallucinated `forEac` when you orginally suggested to use `map`.